debian/nntrainer-dev.install

```diff
@@ -17,6 +17,10 @@
 /usr/include/nntrainer/layer_devel.h
 /usr/include/nntrainer/neuralnet.h
 /usr/include/nntrainer/tensor.h
+<<<<<<< HEAD
+=======
+/usr/include/nntrainer/tensor_dim.h
+>>>>>>> [weight/var_grad] Remove exposure of weight/var_grad
```

Comment:
Could you check this out?

Reply:
I can see updated, maybe deleting this commit?

Reply:
done.